OpenRA.Mods.Common/Traits/Turreted.cs

```diff
@@ -158,6 +158,9 @@ public bool FaceTarget(Actor self, Target target)
 
 		public void StopAiming(Actor self)
 		{
+			if (IsTraitDisabled || IsTraitPaused)
```

Comment:
I'm not convinced that we want to do this when paused.  Can you explain the reasoning here?

Reply:
probably not